Source/WebKit/UIProcess/glib/WebProcessPoolGLib.cpp

```diff
@@ -120,7 +120,7 @@ void WebProcessPool::platformInitializeWebProcess(const WebProcessProxy& process
 
 #if USE(ATSPI)
     static const char* accessibilityBusAddress = getenv("WEBKIT_A11Y_BUS_ADDRESS");
-    parameters.accessibilityBusAddress = accessibilityBusAddress ? String::fromUTF8(accessibilityBusAddress) : WebCore::PlatformDisplay::sharedDisplay().accessibilityBusAddress();
+    parameters.accessibilityBusAddress = accessibilityBusAddress ? String::fromUTF8(accessibilityBusAddress) : WebCore::PlatformDisplay::sharedDisplay().accessibilityBusProxyAddress();
```

Comment:
I think this fails when the sandbox is disabled. Maybe that's why the API tests are failing?

Reply:
But thats the thing we shouldnt want the sandbox disable though. Not unless there is nothing using it at that moment.

Reply:
No, I mean the address will be empty when the sandbox is disabled, because I never set it.

We have to proxy access to the accessibility bus because unrestricted access would be a sandbox escape.